### PR TITLE
[#1786] docs(jdbc): improvement JDBC catalog document

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
1. update PG supported version
2. add warning messages not change gravitino id
3. add not support set MySQL scheme comment message

### Why are the changes needed?

Fix: #1786 #1704 

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
no, just document

### CODE REVIEW

File: docs/jdbc-mysql-catalog.md [hunk 1]
```diff
@@ -51,7 +55,7 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 ### Schema capabilities
 
 - Gravitino schema corresponds to the MySQL database.
-- Supports create schema with comments.
+- Supports create schema, but does not explicitly support setting comments.
```
Comment (yuqi1129) [inline docs/jdbc-mysql-catalog.md:58]: Should we also make changes to the MySQL table?

Comment (FANNG1) [reply docs/jdbc-mysql-catalog.md:58]: mysql table support set comment

File: docs/jdbc-mysql-catalog.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ This software is licensed under the Apache License version 2."
 
 Gravitino provides the ability to manage MySQL metadata.
 
+:::caution
+Gravitino will save some system information in schema and table comment, like `(From Gravitino, DO NOT EDIT: gravitino.v1.uid1078334182909406185)`, please don't change or remove this message.
```
Comment (jerryshao) [inline docs/jdbc-mysql-catalog.md:17]: "Gravitino saves..."

File: docs/jdbc-mysql-catalog.md (hunk 1 above)
Comment (jerryshao) [reply docs/jdbc-mysql-catalog.md:58]: Remove the trailing whitespace.

File: docs/jdbc-postgresql-catalog.md [hunk 3]
```diff
@@ -13,12 +13,16 @@ This software is licensed under the Apache License version 2."
 
 Gravitino provides the ability to manage PostgreSQL metadata.
 
+:::caution
+Gravitino will save some system information in schema and table comment, like `(From Gravitino, DO NOT EDIT: gravitino.v1.uid1078334182909406185)`, please don't change or remove this message.
```
Comment (jerryshao) [inline docs/jdbc-postgresql-catalog.md:17]: Ditto.

File: docs/jdbc-mysql-catalog.md (hunk 1 above)
Comment (FANNG1) [reply docs/jdbc-mysql-catalog.md]: done

File: docs/jdbc-postgresql-catalog.md (hunk 3 above)
Comment (FANNG1) [reply docs/jdbc-postgresql-catalog.md]: done

**Review by jerryshao — APPROVED** [review]

**Pull request merged**